### DISCUSSION TITLE
thoughts/%EB%8F%85%ED%9B%84%EA%B0%90/%EC%82%B0%EB%AC%B8/2024-the-bird-doesnt-look-back/

### DISCUSSION THREAD

**Discussion by giscus** [rlaisqls/blog-comments#3] [Announcements]
# thoughts/%EB%8F%85%ED%9B%84%EA%B0%90/%EB%8F%85%ED%9B%84%EA%B0%90-2024/the-bird-doesnt-look-back/

https://blog.rlaisqls.site/thoughts/%EB%8F%85%ED%9B%84%EA%B0%90/%EB%8F%85%ED%9B%84%EA%B0%90-2024/the-bird-doesnt-look-back/

**Comment by Yoochanhong** [comment]
저도 백준을 놓아버려야할까요.. ㅠ.ㅠ

**Reply by rlaisqls** [reply]
백준을 메트릭을 유지하는 것이 매몰비용에 대한 아쉬움과 더 멀리 비행하기 위한 자기개발 중 어느 쪽에 더 가까운지를 고민해보시면 어떨까요?

**Reply by Yoochanhong** [reply]
아무생각없이 답정너 질문을 해버린 점은 일단 미안하다는 말을 전하고 싶습니다 ㅠㅠ
스트릭을 끊어버린다는건 저한테는 이젠 불가능한 일이 되어버렸네요...
단순히 멋있어보이던 친구를 동경하며 시작한 일이였을뿐인데 어느새 삶의 일부가 되어버린 나머지..
그래도 감수하고 살아가려고 합니다. ㅎㅎ 결별할수 없는 과거는 날개에 매단 돌과 같겠지만 그만큼 날갯짓을 더 하다보면 언젠간 나도 높이 날수 있지 않을까? 라는 생각을 해보며 오늘도 한문제 풀러 가봅니다.
좋은 글 감사합니다 :)

**Reply by rlaisqls** [reply]
돌을 매달고 나는 연습을 하면 날개 근육이 짱 세질 것 같긴 해요.
백준을 푸는 게 현재에 가장 충실하게 집중하는 방법일 수도 있겠죠. 
찬홍님께서 스스로 의미있다고 생각하는 방향으로 선택하신다면 어떤 식으로든 좋은 경험과 자산으로 남을거라 생각합니다. 앞으로도 응원하겠습니다 :)
